atomicapp/nulecule/main.py

```diff
@@ -64,6 +63,7 @@ def __init__(self, app_spec, destination=None,
             destination: where to unpack a nulecule to if it isn't local
             cli_answers: some answer file values provided from cli args
             answers_file: the location of the answers file
+            cli (dict): CLI data
         """
         self.answers = copy.deepcopy(DEFAULT_ANSWERS)
         self.cli_answers = cli_answers
```

Comment:
Ok so I'm going to go out on a limb and say the `self.answers` and `self.cli_answers` should no longer exist as instance/class variables. we should really be creating a `self.config = Config()` in this _init_ function and using that exclusively throughout the rest of the code, right? The point of the Config class is so that we aren't handling our data multiple times in various structures.

We should be able to run `_process_answers()` and update that function to return actual answer data as a return value that we can then use when calling `Config()`. or something along those lines.